src/git.ts

```diff
@@ -100,11 +100,9 @@ export async function deploy(action: ActionInterface): Promise<void> {
   try {
     hasRequiredParameters(action)
 
-    const commitMessage = `${
-      !isNullOrUndefined(action.commitMessage)
-        ? action.commitMessage
-        : `Deploying to ${action.branch} from ${action.baseBranch}`
-    } ${process.env.GITHUB_SHA ? `@ ${process.env.GITHUB_SHA}` : ''} 🚀`
+    const commitMessage = !isNullOrUndefined(action.commitMessage)
+      ? (action.commitMessage as string)
+      : `Deploying to ${action.branch} from ${action.baseBranch} @ ${process.env.GITHUB_SHA} 🚀`
```

Comment:
```suggestion
      : `Deploying to ${action.branch} from ${action.baseBranch} ${process.env.GITHUB_SHA ? `@ ${process.env.GITHUB_SHA}` : ''} 🚀`
```

Let's keep the check here!

Reply:
Yeah sure.
It always will be available though, they specified that in the docs.